### DISCUSSION TITLE
Tailwind scales/colors as CSS variables?

### DISCUSSION THREAD

**Discussion by bstro** [tailwindlabs/tailwindcss#11666] [Help]
I am working on a project with a constraint that requires extremely minimal use of developer dependencies. That being the case, I'd still love to use some of Tailwind's features, namely to interact with various Tailwind scales like spacing/sizes, colors, radii, etc, as CSS variables rather than involve the whole Tailwind machinery. Is there any way to access these or easily dump these from Tailwind into a compiled CSS file with plain old CSS variables?

Thanks!

**Comment by wongjn** [comment]
You could look at copying https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/config.full.js and transforming the data for your needs.
